### ISSUE TITLE
Create dataset loader for UIT-ViCTSD

### ISSUE THREAD

**Issue by SamuelCahyawijaya** [SEACrowd/seacrowd-datahub#271]
Dataloader name: `uit_victsd/uit_victsd.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?uit-victsd

|     Dataset| uit_victsd |
|-------------|---|
| Description | The UIT-ViCTSD (Vietnamese Constructive and Toxic Speech Detection dataset) is a compilation of 10,000 human-annotated comments intended for constructive and toxic comments detection.  Divided into three parts, it includes a Training set with 7,000 comments, a Validation set containing 2,000 comments, and a Test set with 1,000 comments. The composition of the dataset spanning 10 domains, reflecting the diverse expressions found in social media interactions among Vietnamese users. |
| Subsets | constructiveness, toxicity |
| Languages | vie |
| Tasks | Text Classification |
| License | Unknown (unknown) |
| Homepage | https://github.com/tarudesu/ViCTSD |
| HF URL | https://huggingface.co/datasets/tarudesu/ViCTSD |
| Paper URL | https://link.springer.com/chapter/10.1007/978-3-030-79457-6_49 |

**Comment by elyanah-aco** [comment]
#self-assign

**Comment by elyanah-aco** [comment]
@holylovenia @SamuelCahyawijaya @sabilmakbar 

Almost done with dataloader, just making sure if `INTENT_CLASSIFICATION` is the correct task for the `constructiveness` subset?

**Comment by sabilmakbar** [comment]
Yeah `INTENT_CLASSIFICATION` should be fine for constructiveness. For toxicity, we have new type of task called `ABUSIVE LANGUAGE PREDICTION` (https://github.com/SEACrowd/seacrowd-datahub/pull/299).

cc @holylovenia @SamuelCahyawijaya
